app/views/browse/show.html.erb

```diff
@@ -45,9 +45,12 @@
             font_size: "m"
           } %>
           <ul class="govuk-list govuk-!-margin-bottom-7">
-            <% action_link_data(page.slug).each do |link| %>
-              <li>
-                <%= render "shared/browse_action_link", locals: link %>
+            <% each_popular_link_for_slug(page.slug) do |link, analytics_label, link_index, link_count| %>
+              <li
+                  data-module="ga4-link-tracker"
+                  data-ga4-track-links-only=""
+                  data-ga4-link='{"event_name":"navigation","type":"action","index_link":<%= link_index %>,"index_total":<%= link_count %>,"text":"<%= analytics_label %>"}'>
+                <%= render partial: "shared/browse_action_link", locals: {link:} %>
```

Comment:
If we create a `link` with a `data attributes` key this goes back to `<%= render partial: "shared/browse_action_link", locals: {link:} %>`

Reply:
I've removed the partial as it's no longer necessary.

Reply:
Please, see my other comments.